runtests.py

```diff
@@ -79,6 +79,8 @@
          MYPYC_EXTERNAL,
          MYPYC_COMMAND_LINE,
          ERROR_STREAM]),
+    # Test cases to run in typeshed CI
+    'typeshed-ci': 'pytest -k "%s"' % ' or '.join([CMDLINE, EVALUATION, SAMPLES, TYPESHED]),
```

Comment:
Also update `DEFAULT_COMMANDS` below to not include this (or maybe get rid of it, since it's super error-prone, and use a flag here or something).